<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreditCalcResult</class>
 <widget class="QMainWindow" name="CreditCalcResult">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Credit Calc Result</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: #3d3d3d;</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>621</width>
      <height>521</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>3</number>
     </property>
     <property name="leftMargin">
      <number>3</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>3</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horlLayout_MP">
       <item>
        <widget class="QLabel" name="label_MP">
         <property name="styleSheet">
          <string notr="true">width: 40px;
height: 45px;
border: 3px solid rgb(12, 0, 0);
border-radius: 20px;
font: bold 20px;
color: white;
background: gray;
font-family: &quot;Comic Sans MS&quot;;</string>
         </property>
         <property name="text">
          <string>Monthly payment:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_resMP">
         <property name="styleSheet">
          <string notr="true">width: 40px;
height: 45px;
border: 3px solid rgb(12, 0, 0);
border-radius: 20px;
font: bold 20px;
color: white;
background: gray;
font-family: &quot;Comic Sans MS&quot;;</string>
         </property>
         <property name="text">
          <string>res</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horLayout_OP">
       <item>
        <widget class="QLabel" name="label_OP">
         <property name="styleSheet">
          <string notr="true">width: 40px;
height: 45px;
border: 3px solid rgb(12, 0, 0);
border-radius: 20px;
font: bold 20px;
color: white;
background: gray;
font-family: &quot;Comic Sans MS&quot;;</string>
         </property>
         <property name="text">
          <string>Overpayment on a credit:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_resOP">
         <property name="styleSheet">
          <string notr="true">width: 40px;
height: 45px;
border: 3px solid rgb(12, 0, 0);
border-radius: 20px;
font: bold 20px;
color: white;
background: gray;
font-family: &quot;Comic Sans MS&quot;;</string>
         </property>
         <property name="text">
          <string>res</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horLayout_TP">
       <item>
        <widget class="QLabel" name="label_TP">
         <property name="styleSheet">
          <string notr="true">width: 40px;
height: 45px;
border: 3px solid rgb(12, 0, 0);
border-radius: 20px;
font: bold 20px;
color: white;
background: gray;
font-family: &quot;Comic Sans MS&quot;;</string>
         </property>
         <property name="text">
          <string>Total payout:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_resTP">
         <property name="styleSheet">
          <string notr="true">width: 40px;
height: 45px;
border: 3px solid rgb(12, 0, 0);
border-radius: 20px;
font: bold 20px;
color: white;
background: gray;
font-family: &quot;Comic Sans MS&quot;;</string>
         </property>
         <property name="text">
          <string>res</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableView" name="tableView">
       <property name="styleSheet">
        <string notr="true">border: 3px solid rgb(12, 0, 0);
border-radius: 20px;
font: bold 20px;
color: black;
background: white;
font-family: &quot;Comic Sans MS&quot;;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
